src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1598,6 +1582,21 @@ async ValueTask<T> Core(int ordinal, CancellationToken cancellationToken)
             await PgReader.EndReadAsync().ConfigureAwait(false);
             return result;
         }
+
+        async Task<T> GetStream()
```

Comment:
Pass `ordinal` and `cancellationToken` as arguments?

Reply:
Done